pkg/drivers/kic/oci/oci.go

```diff
@@ -64,6 +63,23 @@ func CreateContainerNode(p CreateParams) error {
 		"--label", fmt.Sprintf("%s=%s", nodeRoleKey, p.Role),
 	}
 
+	// volume path in minikube home folder to mount to /var
+	hostVarVolPath := filepath.Join(localpath.MiniPath(), "machines", p.Name, "var")
+	if err := os.MkdirAll(hostVarVolPath, 0777); err != nil {
```

Comment:
Is 0777 necessary? The permissions seems extremely broad. Prefer 0700 or 0711 if possible.